index.rst

```diff
@@ -1,61 +1,190 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+.. sectnum::
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Abstract
+========
 
-  Use the following syntax for sections:
+Given the authentication implementation described in DMTN-169_, Butler then has to decide whether to permit an operation on a collection following requirement DMS-REQ-0340 in LSE-61_.
```

Comment:
DMTN-169 says that access to the database will go through a server that performs authorization checks and that pre-signed URLs will be returned to the user to allow authorized access to datasets.  But neither that document nor this one presents an argument that solely determining authorization by collection is sufficient to meet requirement 3 in [DMTN-169 §3.1](https://dmtn-169.lsst.io/#requirements).

Among other things, such an argument would seem to have to say that all operations require a collection and that no datasets can be added to a collection (directly or via collection chaining) unless the user has authorization to access them.

Reply:
I added some language saying that all Butler operations are done via a collection.